### ISSUE TITLE
Security Vulnerabilty test Fails on inject:guice due to guava:guava@27.1-jre

### ISSUE THREAD

**Issue by joviegas** [google/guice#1464]
Issue : Security Scan on our repository fails with below Vulnerability
```
Issues with no direct upgrade or patch: 
|  Information Disclosure [Medium Severity][https://snyk.io/vuln/SNYK-JAVA-COMGOOGLEGUAVA-1015415] in com.google.guava:guava@27.1-jre | introduced by com.google.inject:guice@4.2.3 > com.google.guava:guava@27.1-jre 
| This issue was fixed in versions: 30.0-android, 30.0-jre
```

One of our packages depends on inject:guice , I used snyk.io tool run the Vulnerability test. Fix for this is upgrading guava@27.1-jre to 30.0-android, 30.0-jre

Question : 

1. Do guice has any plans of upgrading [dependency](https://github.com/google/guice/blob/master/pom.xml#L144-L149).

2. If not Upgraded What is the Risk of running the s/w with this guava@27.1-jre and what is the mitigation plan ?

**Comment by joviegas** [comment]
ping

**Comment by GedMarc** [comment]
there is a pr that has been pulled in that updates v5 to 30 / you can close this....

**Comment by joviegas** [comment]
> there is a pr that has been pulled in that updates v5 to 30 / you can close this....



> there is a pr that has been pulled in that updates v5 to 30 / you can close this....

I am sorry GedMarc, I didnot understand..Could you please point me to that PR and the release which I need to upgrade to resolve this.

**Comment by markmarch** [comment]
Closed by #1495
